packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -25,6 +25,15 @@ export interface EcsRunTaskProps extends sfn.TaskStateBaseProps {
    */
   readonly taskDefinition: ecs.TaskDefinition;
 
+  /**
+   * Use the family name of the task definition instead of full ARN.
+   * This can be used when the latest revision from the family
+   * should be used to run the task
+   *
+   * @default false
+   */
+  readonly useTaskDefinitionFamily?: boolean;
```

Comment:
when would we want to use the full ARN? would it be beneficial to just use the latest revision from the family as the default?

Reply:
I agree that it would be beneficial to just use latest revision always because there is no way currently to configure which revision is the active revision in `TaskDefinition` anyway.

Therefore, I have removed this prop itself in favour of the above default.